docs/pages/User/GettingStarted/test.md

```diff
@@ -85,14 +85,32 @@ You should see the pod offloaded to the remote cluster listed as service endpoin
 
 ### Check Service connectivity 
 
-If you have direct connectivity with the cluster (e.g. K3s) from your host:
+First, you have to scrape the IP address of the service:
 
 ```bash
-SVC_IP=$(kubectl get svc -n test-liqo --template={{.spec.clusterIP}})
+SVC_IP=$(kubectl get svc test-liqo -n test-liqo --template={{.spec.clusterIP}})
 echo $SVC_IP
 ```
 
-<!-- TODO:  add a 'curl' command or something like that that shows that the pod returns the expected page. Show that the command is exactly the same in either cases (i.e., when the pod is local, when the pod is remote) -->
+If you have direct connectivity with the cluster (e.g. K3s) from your host, open a browser and connect to the value of `$SVC_IP` or use the following `curl` command:
+
+```
+curl -v $SVC_IP
+```
+
+If you have not direct connectivity, you can fire up a pod and run a curl inside:
+
+```
+kubectl run --image=curlimages/curl tester -n test-liqo -ti --rm -- curl -L $SVC_IP
+```
+
+You can also connect to the service by using the service name, using the Kubernetes DNS service:
+
+```
+kubectl run --image=curlimages/curl tester -n test-liqo -ti --rm -- curl -L http://test-liqo 
+```
+
+
```

Comment:
I would add some commands that show how to deploy the service either locally or remotely, so that the user can play and test that there's no difference in the user experience.